src/citrine/ara/variables.py

```diff
@@ -101,3 +110,232 @@ def __init__(self, *,
         self.name = name
         self.headers = headers
         self.template = template
+
+
+class AttributeByTemplateAfterProcessTemplate(Serializable['AttributeByTemplateAfterProcessTemplate'], Variable):
+    """Attribute of an object that is marked by an attribute template and derives from a process
+    marked by a locally unique object template.
+
+    TODO: define "locally-unique"
```

Comment:
Unique within the scope of the material history that's being converted into the row